packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -158,19 +154,17 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
 
   @Element() el: HTMLCalciteComboboxItemElement;
 
+  @State() hasContent = false;
+
   // --------------------------------------------------------------------------
   //
   //  Lifecycle
   //
   // --------------------------------------------------------------------------
 
   connectedCallback(): void {
+    // todo: refactor: if grandparent is removed, this will not update
     this.ancestors = getAncestors(this.el);
```

Comment:
if grandparent is removed, I think this would get state and incorrect. I don't think it would update because connectedCallback of a grandparent wouldn't trigger anything here.

Reply:
Yeah, we should revisit this. Can you create an issue to refactor this logic and possibly move it up to `combobox`?

Reply:
created #10459